### PR TITLE
feature: implement ContainerStatus and PodSandboxStatus of cri manager

### PR DESCRIPTION
**Pull request by YaoZengzeng**
Signed-off-by: YaoZengzeng <yaozengzeng@zju.edu.cn>

**1.Describe what this PR did**

**2.Does this pull request fix one issue?** 

fixes part of #420 

**3.Describe how you did it**

**4.Describe how to verify it**

Maybe the [cri-tools](https://github.com/kubernetes-incubator/cri-tools) is what you need.

We may use the following method to verify that we can get status of sandboxes and containers:
```
// Now we must ensure the sandbox image has already exists.
[root@pouch-test cri-tools]# pouch pull k8s.gcr.io/pause-amd64:3.0
[root@pouch-test cri-tools]# cat sandbox-config.json
{
    "metadata": {
        "name": "nginx-sandbox",
        "namespace": "default",
        "attempt": 1,
        "uid": "hdishd83djaidwnduwk28bcsb"
    },
    "linux": {
    }
}
[root@pouch-test cri-tools]# crictl runs sandbox-config.json
6ca39d7986fce58e7fd7fabe17e4acf7e5220ed82deca119fe40218ef6803cc4
[root@pouch-test cri-tools]# crictl inspects 6ca39d7986fce58e7fd7fabe17e4acf7e5220ed82deca119fe40218ef6803cc4
{
  "status": {
    "id": "6ca39d7986fce58e7fd7fabe17e4acf7e5220ed82deca119fe40218ef6803cc4",
    "metadata": {
      "attempt": 1,
      "name": "nginx-sandbox",
      "namespace": "default",
      "uid": "hdishd83djaidwnduwk28bcsb"
    },
    "state": "SANDBOX_NOTREADY",
    "createdAt": "2018-01-12T07:52:38.529798965Z",
    "network": null,
    "linux": null,
    "labels": null,
    "annotations": null
  }
}
[root@pouch-test cri-tools]# cat container-config.json 
{
  "metadata": {
      "name": "busybox"
  },
  "image":{
      "image": "docker.io/library/busybox:latest"
  },
  "command": [
      "top"
  ],
  "linux": {
  }
}
[root@pouch-test cri-tools]# crictl create 6ca39d7986fce58e7fd7fabe17e4acf7e5220ed82deca119fe40218ef6803cc4 container-config.json sandbox-config.json
2b1ae5026fc784bd33f17680b39fa7aea2c68b22cb6bdbf2b74ac7172736ef34
[root@pouch-test cri-tools]# crictl inspect 2b1ae5026fc784bd33f17680b39fa7aea2c68b22cb6bdbf2b74ac7172736ef34
{
  "status": {
    "id": "2b1ae5026fc784bd33f17680b39fa7aea2c68b22cb6bdbf2b74ac7172736ef34",
    "metadata": {
      "attempt": 0,
      "name": "busybox"
    },
    "state": "CONTAINER_CREATED",
    "createdAt": "2018-01-12T07:53:07.935952223Z",
    "startedAt": "1970-01-01T00:00:00Z",
    "finishedAt": "1970-01-01T00:00:00Z",
    "exitCode": 0,
    "image": {
      "image": "docker.io/library/busybox:latest"
    },
    "imageRef": "docker.io/library/busybox:latest",
    "reason": "",
    "message": "",
    "labels": null,
    "annotations": null,
    "mounts": null,
    "logPath": ""
  }
}
```

**5.Special notes for reviews**

### CODE REVIEW

**Comment by YaoZengzeng** [comment]
@allencloud PTAL.

**Comment by allencloud** [comment]
Do we add test case in critools? @YaoZengzeng

**Comment by YaoZengzeng** [comment]
@allencloud It's ok for pouch to pass all the basic sandbox and container operation test in cri-tools if the image is ready.

So please to merge this one 😄

File: daemon/mgr/cri_test.go [hunk 1]
```diff
@@ -25,6 +25,22 @@ func TestParseUint32(t *testing.T) {
 	}
 }
 
+func TestToCriTimestamp(t *testing.T) {
```
Comment (allencloud) [inline daemon/mgr/cri_test.go]: Please add some unit test cases to those returning a non-nil cases. 
Maybe in a follow-up.

**Comment by allencloud** [comment]
LGTM
